src/components/login/facebook-login/facebook-login.jsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import PropTypes from "prop-types";
+import FacebookLogin from "react-facebook-login";
+import "../google-facebook-login.scss";
```

Comment:
Move this import to the bottom of other imports.
All CSS style imports need to placed after all modules imports.

Reply:
Okay, thank you, moved.